### PR TITLE
OWLS-103825: Available should be true for admin server only domain

### PR DESCRIPTION
**Pull request by lennyphan**
Condition Available will be true for a Domain where the only server configured to run is the Admin server.

### CODE REVIEW

**Review by rjeberhard — APPROVED** [review]

File: operator/src/main/java/oracle/kubernetes/operator/DomainStatusUpdater.java [hunk 1]
```diff
@@ -857,13 +857,23 @@ private Collection<DomainPresenceInfo.ServerStartupInfo> getServerStartupInfos()
 
         // when the domain start policy is ADMIN_ONLY, the admin server is considered to be an application server.
```
Comment (ankedia) [inline operator/src/main/java/oracle/kubernetes/operator/DomainStatusUpdater.java:858]: The comment is bit misleading after the change.

**Review by ankedia — APPROVED** [review]
LGTM

**Comment by ankedia** [comment]
@rjeberhard @lennyphan - If the domain has an admin server and a cluster with replicas set to 0, will the domain available condition be `True` or `False` in this case? I'm looking into OWLS-103828 and making sure that the cluster available condition is `False` when replicas is `0`. Just want to make sure that I understand the expected behavior of the domain condition in such cases. Thanks.

**Comment by lennyphan** [comment]
Closing this PR as it has been combined with PR [3602](https://github.com/oracle/weblogic-kubernetes-operator/pull/3602)

**Pull request closed without merging**